src/ASM/ASMs3DLag.C

```diff
@@ -1200,6 +1200,20 @@ bool ASMs3DLag::evaluate (const ASMbase* basis, const Vector& locVec,
 }
 
 
+void ASMs3DLag::constrainFace (int dir, bool open, int dof,
+                               int code, char basis)
+{
+  this->ASMs3D::constrainFace(dir, open, dof, code > 0 ? -code : code, basis);
+}
+
+
+void ASMs3DLag::constrainEdge(int lEdge, bool open, int dof,
+                             int code, char basis)
```

Comment:
Inconsistent spacing/indent in function definition (should be like for constrainFace)

Reply:
yeah, I fixed it in the wrong branch. remedied.